service/src/test/java/uk/gov/hmcts/reform/fpl/model/PlacementTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.model.PlacementOrderAndNotices.PlacementOrderAndNoticesType.NOTICE_OF_HEARING;
+import static uk.gov.hmcts.reform.fpl.model.PlacementOrderAndNotices.PlacementOrderAndNoticesType.PLACEMENT_ORDER;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+class PlacementTest {
+
+    @Test
+    void shouldReturnEmptyListWhenOnlyPlacementOrderIsPresentInListOfOrderAndNotices() {
```

Comment:
wrong method name, it returns Placement object and not list
maybe shouldReturnPlacementWithEmptyList...